ci/eval/compare/default.nix

```diff
@@ -114,7 +115,13 @@ let
           # Adds "10.rebuild-*-stdenv" label if the "stdenv" attribute was changed
           ++ lib.mapAttrsToList (kernel: _: "10.rebuild-${kernel}-stdenv") (
             lib.filterAttrs (_: kernelRebuilds: kernelRebuilds ? "stdenv") rebuildsByKernel
-          );
+          )
+          ++ lib.optional (
+            maintainers ? ${githubAuthorId}
+            && lib.all (lib.flip lib.elem maintainers.${githubAuthorId}) (
+              lib.flatten (lib.attrValues maintainers)
+            )
+          ) "11.by: package-maintainer";
```

Comment:
Maybe it'd be a good idea to link the part of the OfBorg source where this logic is lifted from?